zeppelin-web-vue/src/components/Notebook/Create.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div>
+    <a-modal
+        v-model="showDialog"
+        v-bind:title="this.actionLabel + ' Notebook'"
+        onOk="handleOk"
+        :maskClosable="false"
+      >
+      <template slot="footer">
+        <a-button key="back" @click="handleCancel">Cancel</a-button>
+        <a-button key="submit" type="primary" :loading="loading" @click="handleOk">
+          {{ this.actionLabel }}
+        </a-button>
+      </template>
+
+      <a-form layout="vertical">
+        <a-form-item
+          label="Notebook Name"
+        >
+          <a-input placeholder="Enter Notebook Name"  v-model="name"/>
+        </a-form-item>
+
+        <a-form-item
+          v-if="action === 'create'"
+          label="Default Interpreter"
+        >
+          <a-select
+            showSearch
+            style="width: 180px"
+            :defaultValue="interpreters[0]"
+            v-model="defaultInterpreter"
+          >
+            <a-select-option
+              v-for="interpreter in interpreters"
+              v-bind:key="interpreter.id"
+            >
+              {{interpreter.id}}
+            </a-select-option>
+          </a-select>
+        </a-form-item>
+
+        <a-alert message="Use '/' to create folders. Example: /NoteDirA/Note1" type="info" />
+
+        <input type="hidden" v-model="sourceNotebookId" name="sourceNotebookId" value="" />
+
+      </a-form>
+    </a-modal>
+  </div>
+</template>
+
+<script>
+import { EventBus } from '@/services/event-bus'
+
+export default {
+  name: 'CreateNotebook',
```

Comment:
I need clarify several concepts here, `Notebook` means a collection of notes, and `Note` means a collection of paragraphs. So here it is better to use `CreateNote`  I believe

Reply:
Will fix it.

Reply:
Renamed Notebook to Note wherever it was applicable.